bazel/execution/parse.go

```diff
@@ -152,6 +153,48 @@ func ExtractOpFromJson(opBytes []byte) (*longrunning.Operation, error) {
 	return op, nil
 }
 
+// Converts a longrunning.Operation to a Json-encoded []byte. Because an Operation is not natively compatible
```

Comment:
recommend update comment: Because Operation's Result field is not natively compatible...

Reply:
done